src/service/standard-actions-impl.js

```diff
@@ -125,6 +131,21 @@ export class StandardActions {
    * @param {!./action-impl.ActionInvocation} invocation
    * @private
    */
+  handleAmpNavigateTo_(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return;
+    }
+    const url = invocation.args['url'];
+    const expandedUrl = this.urlReplacements_.expandUrlSync(url);
+    const node = invocation.target;
+    const win = (node.ownerDocument || node).defaultView;
+    win.location = expandedUrl;
```

Comment:
Oh I see. Looks like [`window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) has a few additional params and doesn't support opening in new tabs. If it's alright with you, I'd like to defer this until we receive a FR so we can spec it appropriately.